src/main/webapp/error/403Error.html

```diff
@@ -0,0 +1,184 @@
+				</li>
+				<li>
+					<a href="#">Follow me</a>
+				</li>
+			</ul>
+		</nav>
+		<!-- /#sidebar-wrapper -->
+
+		<!-- Page Content -->
+		<div id="page-content-wrapper">
+			<button type="button" class="hamburger is-closed" data-toggle="offcanvas">
+				<span class="hamb-top"></span>
+				<span class="hamb-middle"></span>
+				<span class="hamb-bottom"></span>
+			</button>
+
+		</div>
+		<!-- /#page-content-wrapper -->
+
+	</div>
+
+	<div class="container">
+
+		<div class="navbar-header">
+			<button type="button" class="navbar-toggle" data-toggle="collapse"
+					data-target="#responsive-menu">
+				<span class="sr-only">Menu</span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+			</button>
+			<a class="navbar-brand" href="#">Great<i class="fa fa-rocket fa-lg" aria-hidden="true">
+				<text/>
+			</i>Start</a>
+		</div>
+		<div class="collapse navbar-collapse" id="responsive-menu">
+			<ul class="nav navbar-nav navbar-left">
+				<li class="active"><a href="#">Home</a></li>
+				<li><a href="#">About</a></li>
+				<li><a href="#">Events</a></li>
+				<li><a href="#">Projects</a></li>
+				<li>
+					<form role="search" class="navbar-form navbar-left">
+						<div class="form-group">
+							<input type="text" placeholder="Search" class="form-control">
+						</div>
+					</form>
+				</li>
+			</ul>
+			<ul class="nav navbar-nav navbar-right">
+				<li class="active"><a href="#modal-login" data-toggle="modal">Login</a></li>
+			</ul>
+		</div>
+	</div>
+</div>
+
+<div class="container">
+	<div class="row">
+		<div class="error-template">
+			<h1>Oops!</h1>
+			<h2>403 Forbidden</h2>
+			<div class="error-details">
+				Access to this resource is denied!<br>
+			</div>
+			<div class="error-actions">
+				<a href="./WEB-INF/views/DesignPage.html" class="btn btn-primary">
+					<span class="icon-home icon-white"></span> Take Me Home </a>
+				<a href="./WEB-INF/views/ContactSupport.html" class="btn btn-default">
+					<span class="icon-envelope"></span> Contact Support </a>
+			</div>
+		</div>
+	</div>
+</div>
+</div>
+
+<div class="modal fade" id="modal-login">
+	<div class="modal-dialog">
+		<div class="modal-content">
+			<div class="modal-header">
+				<h4 class="modal-title">Registration
+					<button class="close" type="button" data-dismiss="modal">
+						<i class="fa fa-close"></i>
+					</button>
+				</h4>
+
+			</div>
+			<div class="modal-body">
+				<form action="" method="POST">
+					<div class="form-group">
+						<input type="text" class="form-control" placeholder="login" value="">
+					</div>
+					<div class="form-group">
+						<input type="password" class="form-control" placeholder="password" value="">
+					</div>
+					<button type="submit" class="btn btn-success">Login</button>
+
+					<form action="/registration">
+						<button class="btn btn-primary" type="submit">
+							<i class="fa fa-user-circle-o"></i> Create an account
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)